### ISSUE TITLE
Talkback doesn't read play books with explore by touch.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#214]
```
What steps will reproduce the problem?
1. Launch the Play Books app.
2. Double-tap on a book.
3. Swipe through the book with explore by touch.
What is the expected output? What do you see instead?
I should be able to read the book like any text using basic gestures.
All I'm able to view are the controls in the book if they aren't hidden. If 
they are, then talkback doesn't say much.
What version of the product are you using? On what operating system?
I'm using version 739 updated February 7, 2013.
4.2.2
Please provide any additional information below.


```

Original issue reported on code.google.com by `mcigo...@gmail.com` on 31 Mar 2013 at 2:57

**Comment by GoogleCodeExporter** [comment]
```
I forgot to mention that I'm running TalkBack version 3.3.
```

Original comment by `mcigo...@gmail.com` on 31 Mar 2013 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
Did this work as expected when using a previous version of TalkBack, or 
possibly a previous version of Play Books?
```

Original comment by `alanv@google.com` on 2 Apr 2013 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
No, explore by touch didn't work with previous versions of play books or talk 
back.
```

Original comment by `mcigo...@gmail.com` on 10 Apr 2013 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
The Play Books app is actually self-voicing when they detect a speaking 
accessibility service is enabled, so the typical TalkBack gestures are reserved 
for navigating around the rest of the interface.

For navigating within the book, you can use the two-finger left or right swipe 
to navigate by page.  To pause continuous reading of the book, just double-tap. 
 Double-tap again to resume.
```

Original comment by `caseybur...@google.com` on 29 May 2013 at 7:54
- Changed state: **Invalid**
